docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.icons.scss

```diff
@@ -27,45 +27,56 @@
   }
 }
 
-// Used with the decanter-external-link class to add an external link icon
+// Used alongside the hb-link--inline mixin when an external icon is needed.
+// And within the fa-ext class in block-layout.scss to add an external link icon.
```

Comment:
I am still struggling to understand when this mixin is used vs the one below called `hb-ext-icon-color`. could you add a couple more comments about each of their use cases?